src/components/checkbox/checkbox.component.tsx

```diff
@@ -109,7 +109,7 @@ export const Checkbox = React.forwardRef(
       );
     }
 
-    const inputProps = {
+    const commonInputProps = {
```

Comment:
**thought** (non-blocking): I personally don't mind just sticking with the old name here in `inputProps`. Happy to go with whatever you like though, as this is just my opinion

Reply:
I'm happy to revert it back to `inputProps`. After looking in `master` we have `inputProps` in `RadioButton` so I'll change it back.